hdk-rust/src/api.rs

```diff
@@ -425,38 +404,31 @@ pub fn remove_entry<S: Into<String>>(_entry: HashString, _message: S) -> ZomeApi
     Err(ZomeApiError::FunctionNotImplemented)
 }
 
-/// Consumes two values, the first of which is the address of an entry, `base`, and the second of which is a string, `tag`,
-/// used to describe the relationship between the `base` and other entries you wish to lookup. Returns a list of addresses of other
-/// entries which matched as being linked by the given `tag`. Links are created in the first place using the Zome API function `link_entries`.
-pub fn get_links<S: Into<String>>(base: &HashString, tag: S) -> ZomeApiResult<GetLinksResult> {
+/// Consumes two values, the first of which is the address of an entry, `base`,
+/// and the second of which is a string, `tag`, used to describe the relationship between the `base`
+/// and other entries you wish to lookup. Returns a list of addresses of other entries which matched
+/// as being linked by the given `tag`.
+/// Links are created in the first place using the Zome API function `link_entries`.
+pub fn get_links<S: Into<String>>(base: &HashString, tag: S) -> ZomeApiResult<Vec<HashString>> {
```

Comment:
Do we really want to get rid of a GetLinksResult? I suppose there will be more information that should be available to the zome around links, esp. crud status and DHT saturation etc... I thought these result structs where partly placeholder for where we can add stuff in the (not too distant) future without breaking our API signature. @zippy?

Reply:
Yes, that's what I was expecting, that it would work exactly like get_entry with an options so you can say if you want sources etc.

Reply:
...